Manifest.toml

```diff
@@ -1,275 +1,274 @@
 # This file is machine-generated - editing it directly is not advised
 
-[[Adapt]]
```

Comment:
Not new, but is it a good idea to check in the manifest? 

Will this constrain CI to run these exact versions, or will it resolve to the latest allowed?

Reply:
I think CI will use these versions if possible. I believe the Manifest was originally checked in because Diffractor depended on different branches of some dependencies, but I don't think that's the case anymore. @keno, should we just delete the Manifest?

Reply:
I think I'll merge this as-is and then we can revisit whether we want to just delete the manifest later.